README.md

```diff
@@ -1,11 +1,11 @@
-# Census Event Generator
```

Comment:
Please check the consistency of heading levels throughout this README. They seem a bit haphazard.

Sorry I know many of these problems are inherited, but hopefully since we have changed it we can fix issues.